presto-product-tests/conf/tempto/tempto-configuration-for-docker-default.yaml

```diff
@@ -4,3 +4,8 @@ databases:
   presto:
     host: presto-master
     configured_hdfs_user: hive
+    cli_ldap_truststore_path: /etc/ldap/cacerts.jks
```

Comment:
I'll move it to a separate commit.

Also, if we are going to use another scripting language to do the merge, I'd prefer Python, which should already be installed on most of our dev systems due to its use in presto-admin.

http://stackoverflow.com/questions/6866600/yaml-parsing-and-python